src/resources/common.yml

```diff
@@ -149,6 +149,15 @@ accessType_public:
 accessType_private:
   en: private
   de: privat
+lessonsMode:
+  en: Mode
+  de: Modus
```

Comment:
As "Mod(e|us)" is very generic, it would maybe be clearer if we called it "Bearbeitungsmodus" or "Bearbeitungsberechtigung" (editing permissions?)?

Reply:
Maybe something more towards "lessons" editing, to not create the impression it's about the whole room? 
Initially I had "Unterrichtseinheiten-Modus".

Reply:
Yeah, that would be good, it just results in very long labels (not so much in english):
* "lesson editing mode"
* "Bearbeitungsmodus für Unterrichtseinheiten"
?

Reply:
Thanks, will go with
- "Lessons handling mode" (since it's not just about editing, also adding and removing)
- "Bearbeitungsmodus für Unterrichtseinheiten"

Reply:
Changed. Went with "Unterrichtseinheiten-Bearbeitungsmodus" so that it wraps to 2 rows instead of 3 in the rooms table 😅

Reply:
:laughing: